From 44470e9035d5f72ba2194ac3ff1c2fbe262d56a2 Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Sat, 26 Sep 2020 02:26:57 +0300
Subject: [PATCH 1/3] Add soong namespace for init scripts

* Not all devices can use those, so make them opt-in
* Stop conflicts with every qcom device tree that exists

Change-Id: I7930d9fca954904d1c0b6eed16e65a90b9440405
---
 etc/Android.bp | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/etc/Android.bp b/etc/Android.bp
index e90b151..108e64d 100644
--- a/etc/Android.bp
+++ b/etc/Android.bp
@@ -1,3 +1,6 @@
+soong_namespace {
+}
+
 prebuilt_etc {
     name: "init.qcom.usb.rc",
     src: "init.qcom.usb.rc",
-- 
2.34.0

